docs/RELEASE_CHECKLIST.md

```diff
@@ -41,7 +42,10 @@ Replace `RELEASE_TAG` below with the actual release tag.
       Use an annotated tag (`git tag -a RELEASE_TAG -m RELEASE_TAG`) with the
       annotation being `RELEASE_TAG`.
       Push `RELEASE_TAG` to GitHub.
-- [ ] Wait for CI build for tagged commit to finish.
+- [ ] Wait for CI build for tagged commit to finish (see the list of pipelines at
+      [zivgitlab](https://zivgitlab.uni-muenster.de/pymor/pymor/-/pipelines)).
+- [ ] Merge the automatic PR at [`pymor/docs`](https://github.com/pymor/docs) and
```

Comment:
I would have expected the PR to merge automatically. Is that not the case?

Reply:
The `automerge` label is applied automatically, and I guess the person who pushed the tag is also automatically assigned, but I don't think the merge is automatic (https://github.com/pymor/docs/pull/33).
@HenKlei do you remember how it worked before (https://github.com/pymor/docs/pull/32)?

Reply:
I have to admit that I have no idea.

Reply:
It is [supposed to be automatic](https://github.com/pymor/docs/blob/main/.github/workflows/automerge_update.yml) but for some reason the action does not seem to fire. Maybe we simplify things by removing the automation (attaching the label and auto merging) and live with the added manual labor?

Reply:
I found that the "Allow auto-merge" setting was not enabled in the `pymor/docs` repo. It should be on now.

I'm not sure if it will make things work, but it might be easier to use the [`enable-pyll-request-automerge` action](https://github.com/marketplace/actions/enable-pull-request-automerge) in [here](https://github.com/pymor/docs/blob/535fa770d2317cab7c0ca38d10146b5b70db9be9/.github/workflows/update_for_release.yml) instead of using labels.